openslides/core/static/js/core/base.js

```diff
@@ -476,6 +476,12 @@ angular.module('OpenSlidesApp.core', [
 ])
 
 // Template hooks
+// 2 possible uses:
+// - { Id: 'myHookId', template: '<button>click me</button>' }
+// - { Id: 'myHookId', templateUrl: '/static/templates/plugin_name/my-hook.html' }
```

Comment:
Ok, now I Know: All of our templates are cached via gulp in the template cache and I would expect any plugin also to to so. See gulpfile line 89: Any plugin should transform all templates to JS and use the hooks inside app.py to add these JS-files to the client. Then you do not need to do this hot reloading bay hand here. AngularJS should just know the template under its path.